### DISCUSSION TITLE
QR Code for link

### DISCUSSION THREAD

**Discussion by andytristram** [pglombardo/PasswordPusher#1588] [Ideas]
Hi,

I'm not sure if it would be useful for us, but generating a QR code for the link might be handy.
Like transfer.pw does.

Thanks,
Andy

**Comment by pglombardo** [comment]
I think it's a great idea @andytristram.  I hope to get to this soon.

**Comment by pglombardo** [comment]
Hi @andytristram - just a heads up that I've added QR code support in #2037 and will be releasing it soon.  I'll update here once it out.

**Comment by pglombardo** [comment]
This has been released in [v1.40.0](https://github.com/pglombardo/PasswordPusher/releases/tag/v1.40.0).  Thanks for the suggestion!

New QR codes on the preview page

![Screenshot 2024-04-10 at 13 32 56](https://github.com/pglombardo/PasswordPusher/assets/395132/db1fbf8d-4d2e-4f3d-b9ce-a67c5f0185ff)


And the printable/shareable page:

![Screenshot 2024-04-10 at 13 33 02](https://github.com/pglombardo/PasswordPusher/assets/395132/0672070f-ba7d-49a4-866f-481c757ce853)
